dist/docker/Readme.md

```diff
@@ -1,68 +1,71 @@
-# Docker Container Name
+# qBittorrent-nox Docker Image
 
-This Dockerfile allows you to build a qBittorrent-nox container
+This Dockerfile helps you to build a docker image containing qBittorrent-nox
 
-## Getting Started
+## Prerequisities
 
-### Prerequisities
+In order to build/run this image you'll need docker installed: https://docs.docker.com/get-docker/
 
+## Building docker image
 
-In order to run this container you'll need docker installed.
-
-* [Windows](https://docs.docker.com/windows/started)
-* [OS X](https://docs.docker.com/mac/started/)
-* [Linux](https://docs.docker.com/linux/started/)
-
-
-## Built
-
-in the docker folder run
-
-```shell
-release="4.2.0" ; sudo docker build --build-arg BUILD_TYPE=Release --build-arg RELEASE=$release -t qbittorrent-nox:$release --rm  .
-```
-
-where:
-
-* the `release` variable is the specific tagged version you want to build
-* the `BUILD_TYPE` argument is the build you want to create `Debug` or `Release`
-* the `RELEASE` argument works as the but is the actual argument given to docker, in the above script is defined by the `release` variable
-
-
-### Usage
-
-#### Container Variables
-
-there is one important variable to run the container:
-
-* the `LEGAL` varible defines if you accept the Legal Notice, put accept as a value only if you understand and accept the Legal Notice
-
-#### Volumes
-
-there are three main locations:
-
-* `config` contains qBittorrent configurations
-* `data` contains qBittorrent application data
-* `downloads` contains the files downloaded by qBittorrent
-
-```shell
-docker run give.example.org/of/your/container:v0.2.1 parameters
-```
-
-#### Network
-
-on the port `8080` the webinterface is run
-
-#### RUN
-
-To start the the docker image simply run
-
+In this docker folder run:
 ```shell
-docker run --env LEGAL=accept -p 8080:8080 -v /your/path/config:/config -v /your/path/data:/data -v /your/path/download:/downloads --name qBittorrent qbittorrent-nox:4.2.0
+export \
+  qbtVersion=git
+docker build \
+  --build-arg QBT_VERSION="$qbtVersion" \
+  -t qbittorrent-nox:"$qbtVersion" \
+  .
 ```
 
-to stop the container
-
-```shell
-docker stop qBittorrent
-```
+### Parameters
+
+* `qbtVersion`
+  This environment variable specifies the version of qBittorrent-nox to be built. \
+  For example, `4.4.0` is a valid entry. You can find all tagged versions [here](https://github.com/qbittorrent/qBittorrent/tags). \
+  Or you can put `git` to build the latest development version.
+
+## Running docker image
+
+* To start the the docker image simply run:
+  ```shell
+  export \
+    qbtVersion=git \
+    qbtEULA=accept \
+    qbtWebUIPort=9000
```

Comment:
nit: I would expect these env vars to be upper camel case (i.e. `QBT_VERSION`)

Reply:
👍